server-common/src/test/java/org/apache/kafka/server/common/FeaturesTest.java

```diff
@@ -94,7 +94,7 @@ public void testInvalidValidateVersion() {
         assertThrows(IllegalArgumentException.class,
              () -> Features.validateVersion(
                  TestFeatureVersion.TEST_2,
-                 Collections.singletonMap(MetadataVersion.FEATURE_NAME, MetadataVersion.IBP_3_7_IV0.featureLevel())
+                 Collections.singletonMap(MetadataVersion.FEATURE_NAME, MetadataVersion.IBP_3_6_IV0.featureLevel())
```

Comment:
Why do we need to change to 3.6?

Reply:
no reason, I think this is stale. I will revert it back